pkg/bff/gds.go

```diff
@@ -221,6 +246,15 @@ func (s *Server) VerifyContact(c *gin.Context) {
 }
 
 // Returns the user's current registration form if it's available
+//
+// @Summary Get the user's current registration form [read:vasp]
+// @Description Get the registration form associated with the user's organization.
+// @Tags registration
+// @Produce json
+// @Success 200 {object} object "Registration form"
```

Comment:
Swagger wasn't able to follow the external protobuf dependencies so for right now I had to specify them as generic objects.